networking/determine-best-networking.md

```diff
@@ -1,110 +1,219 @@
+
+### {{site.prodname}} compatible CNI plugins and cloud provider integrations
+In addition to the {{site.prodname}} CNI plugins and built in networking modes, {{site.prodname}} is also compatible with a number of third party CNI plugins and cloud provider integrations.
+
+**Amazon VPC CNI**
+
+The Amazon VPC CNI plugin allocates pod IPs from the underlying AWS VPC and uses AWS elastic network interfaces to provide VPC native pod networking (pod IPs that are routable outside of the cluster). It is the default networking used in {% include open-new-window.html text='Amazon EKS' url='https://aws.amazon.com/eks/' %}, with Calico for network policy enforcement.
 
 **Azure CNI**
 
-The Azure CNI plugin configures the Azure virtual network to provide pod networking. It is the default networking used in Microsoft AKS, with {{site.prodname}} for [network policy enforcement]({{ site.baseurl }}/security/calico-network-policy).
+The Azure CNI plugin allocates pod IPs from the underlying Azure VNET configures the Azure virtual network to provide VNET native pod networking (pod IPs that are routable outside of the cluster). It is the default networking used in {% include open-new-window.html text='Microsoft AKS' url='https://azure.microsoft.com/en-us/services/kubernetes-service/' %}, with Calico for network policy enforcement.
+
+**Google cloud provider**
+
+The Google cloud provider integration uses host-local IPAM CNI plugin to allocate pod IPs and programs the Google cloud network Alias IP ranges to provide VPC native pod networking on Google cloud (pod IPs that are routable outside of the cluster). It is the default for Google Kubernetes Engine (GKE), with Calico for network policy enforcement.
+
+**Host local IPAM**
+
+The host local CNI IPAM plugin is a commonly used IP address management CNI plugin, which allocates a fixed size IP address range (CIDR) to each node, and then allocates pod IP addresses from within that range.  The default address range size is 256 IP addresses (a /24), though 2 of those IP addresses are reserved for special purposes and not assigned to pods. The simplicity of host local CNI IPAM plugin makes it easy to understand, but results in less efficient IP address space usage compared to Calico CNI IPAM plugin.
 
 **Flannel**
 
-Flannel routes pod traffic using static per-node CIDRs. It provides a number of networking backends. [Calico can be used for network policy enforcement]({{ site.baseurl }}/security/calico-network-policy).
+Flannel routes pod traffic using static per-node CIDRs obtained from the host-local IPAM CNI plugin. Flannel provides a number of networking backends, but is predominantly used with its VXLAN overlay backend. {{site.prodname}} CNI and {{site.prodname}} network policy can be combined with Flannel and the host-local IPAM plugin to provide a VXLAN network with policy enforcement.  This combination is sometimes referred to as “Canal”.  
```

Comment:
Casey convinced me to start using "flannel" lowercase so I've been going with that.